styles/styles.css

```
@@ -1584,10 +1632,11 @@ body {
   margin-bottom: 32px;
   font-family: "PT Sans", sans-serif;
```

```suggestion
  font-family: inherit;
```
вообще везде где можно)